include/deal.II/lac/arpack_solver.h

```diff
@@ -238,6 +246,17 @@ class ArpackSolver : public Subscriptor
   bool initial_vector_provided;
   std::vector<double> resid;
 
+  /**
+   * Real part of the shift
+   */
+  double sigmar;
```

Comment:
I think it might be nicer to store the real and imaginary parts as a `std::complex<double>`. Same for the parpack interfaces.